ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -509,7 +509,7 @@ else
     <br />
     <div class="centerwrapper">
         <div class="inlineblockwrapper">
-            <div id="submissionHistory" class="hideablepanel">
+            <div id="submissionHistory" style="display: none" class="hideablepanel">
```

Comment:
This won't be as simple as just hiding by default.  If you navigate to the page and there are already submissions, we do need to show the panel upon first load.  So, you'll want to remove this style tag here, and instead add some JS to optionally add the display:none if there are no submissions.  Line 626 right after the code to make the panel open and close should be a good place to do that.

Reply:
Oh yeah I had the right model in mind but I skipped a step when implementing. I meant to only add display:none if the model knows there are no submissions; then I'm guaranteed that the page won't jerk. I'll wrap this line in the appropriate check...